packages/core/src/lib/toast/toast-container.svelte

```diff
@@ -28,14 +27,13 @@ $: pageIsVisible.set(visibilityState === 'visible');
 <div
   role="status"
   aria-label="Toasts"
-  class="pointer-events-auto fixed bottom-0 left-1/2 top-auto z-max -translate-x-1/2 transform overflow-hidden p-1.5"
+  class="pointer-events-auto fixed bottom-0 left-1/2 top-auto z-max -translate-x-1/2 transform overflow-hidden pb-6"
 >
   <ul class="flex flex-col items-center gap-2">
```

Comment:
`gap-2` (8px) is wrong here

<img width="437" alt="Screenshot 2023-12-26 at 09 36 32" src="https://github.com/viamrobotics/prime/assets/2963448/c8928f2e-0c47-4f4a-ac77-e307afbde85d">

12px is `gap-3`